packages/spec_cli/lib/src/command_runner.dart

```diff
@@ -231,7 +231,12 @@ Future<int> spec({
         final sub = ref.listen($exitCode.future, (previous, current) {});
 
         // Outside of watch mode, quit as soon we know what the exit code should be
-        return sub.read();
+        return () {
```

Comment:
This function block is unnecessary. You don't have to put everything on the `return` keyword

Reply:
I rewrote it a bit.